src/stories/modal-pause/modal-pause.scss

```diff
@@ -0,0 +1,75 @@
+$MODAL_PAUSE_CONTAINER_WIDTH: 550px;
```

Comment:
Mmm. We are already not consistent with this, because we use caps some places and not others, but maybe for a new component I would just make it not capitalized.

Reply:
I have fixed it 🙌